hw_diag/views/diagnostics.py

```diff
@@ -32,6 +32,7 @@
 from hw_diag.utilities.security import GnuPG
 from hw_diag.utilities.auth import authenticate
 from hw_diag.utilities.diagnostics import read_diagnostics_file
+from hw_diag.utilities.balena_supervisor import BalenaSupervisor
 
 
 logging.basicConfig(level=os.environ.get("LOGLEVEL", "DEBUG"))
```

Comment:
Shall we add `@authenticate` to the /json endpoint as well as it's not needed for production

Reply:
I meant on line 44